.azure-pipelines/guardian/SDL/.gdnsuppress

```diff
@@ -0,0 +1,115 @@
+{
+  "hydrated": false,
+  "properties": {
+    "helpUri": "https://eng.ms/docs/microsoft-security/security/azure-security/cloudai-security-fundamentals-engineering/security-integration/guardian-wiki/microsoft-guardian/general/suppressions",
+    "hydrationStatus": "This file does not contain identifying data. It is safe to check into your repo. To hydrate this file with identifying data, run `guardian hydrate --help` and follow the guidance."
+  },
+  "version": "1.0.0",
+  "suppressionSets": {
+    "default": {
+      "name": "default",
+      "createdDate": "2024-02-06 20:37:57Z",
+      "lastUpdatedDate": "2024-02-06 20:37:57Z"
+    }
+  },
+  "results": {
+    "de854c6ab9b27b1ec642cec1a51059b92f88815a231c1c8f4b8e71d9e378f174": {
+      "signature": "de854c6ab9b27b1ec642cec1a51059b92f88815a231c1c8f4b8e71d9e378f174",
+      "alternativeSignatures": [
+        "79827cf2e75a7f99eec716562fb9fa0d49014ac96b7afc76d29d79a33e8edd94"
+      ],
+      "memberOf": [
+        "default"
+      ],
+      "createdDate": "2024-02-06 20:37:57Z"
+    },
+    "ebe0026db71ae4bb2d8e76b706198003079a5c095658de8dd49e67f7e572ea46": {
+      "signature": "ebe0026db71ae4bb2d8e76b706198003079a5c095658de8dd49e67f7e572ea46",
+      "alternativeSignatures": [],
+      "memberOf": [
+        "default"
+      ],
+      "createdDate": "2024-02-06 20:37:57Z"
+    },
+    "bb02ef965dce40b6c950b6d0cf8a9f41c08c5e33f17268fb208f3687b11ffa26": {
+      "signature": "bb02ef965dce40b6c950b6d0cf8a9f41c08c5e33f17268fb208f3687b11ffa26",
```

Comment:
I have no idea what these hashes mean and how to review them. I trust you Max to do it right.

Reply:
Please add a comment explaining how this file was created and when we should update it.

Reply:
Updated the main PR description.